src/psyclone/psyir/tools/dependency_tools.py

```diff
@@ -406,9 +407,11 @@ def can_loop_be_parallelised(self, loop, loop_variable=None,
 
         :param loop: the loop node to be analysed.
         :type loop: :py:class:`psyclone.psyir.nodes.Loop`
-        :param str loop_variable: Optional name of the variable that is\
-                                  parallelised. If not specified, the loop\
-                                  variable of the loop is used.
+        :param loop_variable: Optional symbol of the variable that is \
+            parallelised. If not specified, the loop variable of the loop \
+            is used.
+        :type loop_variable: \
+            :py:class:`psyclone.psyir.symbol.datasymbol.DataSymbol`
```

Comment:
Just `psyclone.psyir.symbols.DataSymbol` I think?

Reply:
Ah yes. I was actually wondering about this in some other cases, but can't remember where now ... might have been different PR.